src/screens/AdminPortal/OrganizationEvents/OrganizationEvents.tsx

```
@@ -140,38 +148,105 @@ function organizationEvents(): JSX.Element {
-  const {
-    data: orgData,
-    loading: orgLoading,
-    error: orgDataError,
-  } = useQuery(GET_ORGANIZATION_DATA_PG, {
-    variables: {
-      id: currentUrl,
-      first: 10,
-      after: null,
+  const { data: orgData } = useQuery(
+    GET_ORGANIZATION_DATA_PG,
+    {
```

ESLint

[error] Replace `⏎····GET_ORGANIZATION_DATA_PG,⏎···` with `GET_ORGANIZATION_DATA_PG,`

(prettier/prettier)